src/expand.rs

```diff
@@ -265,6 +265,14 @@ fn run_tests<I, S>(
     }
 }
 
+fn matches_expected(expected: &str, actual: &str) -> bool {
+    actual.contains(expected) || trim_each_line(actual).contains(&trim_each_line(expected))
+}
+
+fn trim_each_line(input: &str) -> String {
```

Comment:
Collecting the lines like this means there is not more line separation. So the following two will match:

```
something1
somethign2
```
```
something1something2
```
We probably don't want that. Maybe you can re-introduce newlines alone, such that the lines are matched against each other exclusively

Reply:
oh you're absolutely right. I think I *may* have rushed this a bit